StorageAccessAPI/explainer.md

```diff
@@ -2,7 +2,7 @@
 
 ## Status of this Document
 This document is intended as a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
-* This document status: **Active**
+* This document status: **Archived**
```

Comment:
What's the difference between **Archived** and _Archived_?

Reply:
Most are bold, a few are italics. For this PR I just stuck with whatever the previous author had.